openfecwebapp/templates/macros/cycle-select.html

```diff
@@ -1,4 +1,4 @@
-{% macro cycle_select(cycles, cycle=none, duration=2, location='query', id='cycle-select', class='') %}
+{% macro cycle_select(cycles, cycle=none, duration=2, location='query', id='cycle-select', class='', show_two_year_period=False) %}
```

Comment:
This isn't quite right, but I think I did a bad job explaining. It should show all two-year periods, not just the election years. So for a candidate who ran in 2010 and 2016, it should show: 2009-2010, 2011-2012, 2013-2014, 2015-2016. This set of years is available in the `cycles` template var. 

The `committee_cycle_select` macro below does this for committee pages, so maybe it makes sense to repurpose it for this case. 

I'm down to pair on this too, as the logic gets real wonky real fast.